### ISSUE TITLE
Could you share the data of FIGER?

### ISSUE THREAD

**Issue by zig-kwin-hu** [limteng-rpi/fet#5]
Hello, I hope to compare with your method on FIGER(Gold), but I don't know how you divide the training set and validation set. Could you please share your processed data?

**Comment by limteng-rpi** [comment]
Hi, I'm closing this issue as we've communicated via email
